src/ui/public/directives/rows.js

```diff
@@ -40,12 +40,7 @@ module.directive('kbnRows', function ($compile, $rootScope, getAppState, Private
         let $cellContent;
 
         if (contents instanceof AggConfigResult) {
-          const isCellContentFilterable =
-            contents.type === 'bucket'
-            && contents.aggConfig.getField()
-            && contents.aggConfig.getField().filterable;
-
-          if (isCellContentFilterable) {
+          if (contents.type === 'bucket') {
```

Comment:
Ran into a couple issues but I think they have an easy fix.

* This adds filter icons to agg buckets that don't support filtering, like geohash_grid:

![screen shot 2017-05-05 at 4 07 26 pm](https://cloud.githubusercontent.com/assets/6239176/25764047/0bd79928-31b4-11e7-9ed0-3a9e236fa05f.png)

We can check whether an agg type supports filtering with the following condition:

```
_.get(contents, 'aggConfig.type.createFilter')
```

* This also adds filter icons to fields that are aggregatable, but not filterable, which will thrown an error if you try to filter on them. It's probably not common, but it's possible if you had a mapping like this:

```
PUT aggnofilter
{
  "mappings": {
    "aggnofilter": {
      "properties": { 
      "notindexed": {
        "type": "integer",
        "index": false
      }
      }
    }
  }
}
```

We can also check for this with the following condition:

```
(contents.aggConfig.getField() ? contents.aggConfig.getField().filterable : true);
```

Putting it altogether, we can write this condition as follows:

```
const isCellContentFilterable =
  contents.type === 'bucket'
  && _.get(contents, 'aggConfig.type.createFilter')
  && (contents.aggConfig.getField() ? contents.aggConfig.getField().filterable : true);
```